### PR TITLE
Adding TAC instructions

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8853

### CODE REVIEW

File: Teams/emergency-calling-dispatchable-location.md [hunk 1]
```diff
@@ -75,6 +75,20 @@ The following table shows the types of emergency addresses and associated routin
 
 ## Enable end users to configure their emergency address
 
+### Using the Microsoft Teams admin center
+
+1. In the left navigation of the Microsoft Teams admin center, go to **Voice** > **Emergency policies**.
+2. Click **Add**.
```
Comment (JohanFreelancer9) [inline Teams/emergency-calling-dispatchable-location.md:81]:
```suggestion
2. Select **Add**.
```

File: Teams/emergency-calling-dispatchable-location.md [hunk 2]
```diff
@@ -75,6 +75,20 @@ The following table shows the types of emergency addresses and associated routin
...
+3. Enter a Name for the emergency calling policy, for example "E911WFH".
```
Comment (JohanFreelancer9) [inline Teams/emergency-calling-dispatchable-location.md:82]:
```suggestion
3. Enter a name for the emergency calling policy, for example, "E911WFH".
```

File: Teams/emergency-calling-dispatchable-location.md [hunk 3]
```diff
@@ -75,6 +75,20 @@ The following table shows the types of emergency addresses and associated routin
...
+4. Turn on **External location lookup mode**.
+5. Click **Apply**.
```
Comment (JohanFreelancer9) [inline Teams/emergency-calling-dispatchable-location.md:84]:
```suggestion
5. Select **Apply**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**